frontend/package.json

```diff
@@ -11,10 +11,13 @@
     "lint:check": "eslint --ext \".ts,.vue\" --ignore-path .gitignore .",
     "lint:fix": "eslint --ext \".ts,.vue\" --ignore-path .gitignore --fix .",
     "format:check": "prettier --check .",
-    "format:fix": "prettier --write --list-different ."
+    "format:fix": "prettier --write --list-different .",
+    "fl": "prettier --write --list-different . && eslint --ext \".ts,.vue\" --ignore-path .gitignore --fix ."
```

Comment:
What is the purpose of this?

Reply:
I used it to format and lint at the same time while working. It can be deleted.